pkg/daemon/daemon.go

```diff
@@ -1574,18 +1574,14 @@ func MaybePersistNetworkInterfaces(osRoot string) error {
 
 	if hostos.IsEL8() {
 		glog.Info("Persisting NIC names for RHEL8 host system")
-	} else {
-		// If we're not on RHEL, just output the current state.  We don't strictly need to do
-		// this but it will greatly help debugging and validation.
-		cmd.Args = append(cmd.Args, "--inspect")
```

Comment:
Wait, but this will make us persist the NIC names even when we're starting from rhel9, right?

Reply:
I intended the cmd logic below so that it's under the el8 check now. :)